.github/workflows/linux-node.js-functionapp-on-azure.yml

```
@@ -0,0 +1,59 @@
+name: Deploy Node.js project to Azure Function App
+
+on:
+  [push]
+
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+#
+# 1. Set up the following secrets in your repository:
+#   AZURE_FUNCTIONAPP_PUBLISH_PROFILE
+#
+# 2. Change these variables for your configuration:
+env:
+  AZURE_FUNCTIONAPP_NAME: kim-spike-af    # set this to your application's name
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: '.'      # set this to the path to your web app project, defaults to the repository root
+  NODE_VERSION: '12.x'                     # set this to the node version to use (supports 8.x, 10.x, 12.x)
+  DOTNET_VERSION: '3.1.100'              # set this to the dotnet version to use
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: 'Checkout GitHub Action'
+      uses: actions/checkout@master
+
+    - name: Setup Node ${{ env.NODE_VERSION }} Environment
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+
+    - name: Setup DotNet ${{ env.DOTNET_VERSION }} Environment
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+    
+    - name: 'Setup Durable Functions & Function Core Tools'
+      shell: bash
+      run: |
+        npm install -g azure-functions-core-tools@3
+        func extensions install --package Microsoft.Azure.WebJobs.Extensions.DurableTask --version 2.1.0
```

We also want Cosmos:

```suggestion
        func extensions install --package Microsoft.Azure.WebJobs.Extensions.DurableTask --version 2.1.0
        func extensions install --package Microsoft.Azure.WebJobs.Extensions.CosmosDB --version 3.0.5
```